cmd/nitro/nitro.go

```diff
@@ -1035,3 +1034,11 @@ func NewLiveNodeConfig(args []string, config *NodeConfig) *LiveNodeConfig {
 		config: config,
 	}
 }
+
+type LiveNodeConfigFetcher struct {
+	*LiveNodeConfig
+}
+
+func (f *LiveNodeConfigFetcher) Get() *arbnode.Config {
```

Comment:
alternatively, if we would rename Get interface method to e.g. GetNodeConfig, then it could be implemented directly for LiveNodeConfig

Reply:
I think this is a good idea